includes/classes/SiteAutomation/Request.php

```diff
@@ -254,18 +273,33 @@ private function request( $timeout ) {
 	/**
 	 * Process response from Sophi.
 	 *
-	 * @param array $response Response of Site Automation API.
-	 * @param bool  $bypass_cache Whether to bypass cache or not.
+	 * @param array    $response Response of Site Automation API.
+	 * @param bool     $bypass_cache Whether to bypass cache or not.
+	 * @param int|bool $post_id The post id to update the post meta with response or false.
 	 *
 	 * @return array
 	 */
-	private function process( $response, $bypass_cache ) {
+	private function process( $response, $bypass_cache, $post_id ) {
 		if ( ! $response ) {
 			return [];
 		}
 
-		if ( ! $bypass_cache ) {
-			update_option( "sophi_site_automation_data_{$this->page}_{$this->widget}", $response );
+		if ( ! $bypass_cache && post_type_exists('sophi-response') ) {
```

Comment:
Can we make `post_type_exists('sophi-response')` to `post_type_exists( 'sophi-response')`? :)